### PR TITLE
Pull eth_bots out of examples

### PR DESCRIPTION
**Pull request by dpaiton**

### CODE REVIEW

File: eth_bots/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    
+    <aside>
+    💡 **NOTE:** This command overwrites your `.env` file, so make sure you're ok with losing whatever contents are in it first!
+    
+    ```bash
+    python eth_bots/populate_env.py 0xUSER_PRIVATE_KEY > .env
+    ```
+    
+    This will generate new environment variables for the bots and write them to the `.env` file.
+    The new variables are private keys, Base budgets, and Eth budgets for all of the agents you specified in your config.
```
Comment (wakamex) [inline eth_bots/README.md:78]:
```suggestion
    The new variables are private keys, randomly generated Base and Eth budgets (which you can manually overwrite) for all of the agents you specified in your config.
```

File: eth_bots/main.py [hunk 2]
```diff
@@ -6,8 +6,8 @@
 
 from eth_typing import BlockNumber
 
-from examples.eth_bots.setup_experiment import setup_experiment
-from examples.eth_bots.trade_loop import trade_if_new_block
+from eth_bots.setup.setup_experiment import setup_experiment
+from eth_bots.setup.trade_loop import trade_if_new_block
```
Comment (wakamex) [inline eth_bots/main.py:10]: `trade_loop` doesn't seem like a `setup` thing. maybe `core` is a better descriptor?

File: eth_bots/README.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+We have set up bots to run on a dedicated AWS EC2 instance.
+These instructions can likely be followed for other unix/linux setups, however.
+
+## Step 1: Install elfpy
+
+1. make a fork of the elf `delvtech/elf-simulations` repo ([instructions](https://docs.github.com/en/get-started/quickstart/fork-a-repo?tool=webui&platform=mac)).
+
+    >**💡NOTE:** 
+    >You’ll probably want to configure the remote (`delvtech`) repository with your fork. Once you’ve made the fork and cloned it, navigate inside (`cd elf-simulations`), and run
```
Comment (wakamex) [inline eth_bots/README.md:11]:
```suggestion
    >If you want to save your changes, [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the remote (`delvtech`) and clone your fork instead. Then, navigate inside (`cd elf-simulations`), and run
```

File: eth_bots/README.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    >git merge upstream/main
+    >```
+    >
+
+    
+2. Install elfpy by following the installation instructions found [on github](https://github.com/delvtech/elf-simulations/blob/main/INSTALL.md).
+    
+    >**💡NOTE:**
+    >
+    >**pyenv:** If your the AWS server is running bash on linux, then to install Pyenv you’ll want to follow the “[automatic installer](https://github.com/pyenv/pyenv#automatic-installer)” instructions, which tells you to run `curl https://pyenv.run | bash` in your AWS terminal.
```
Comment (wakamex) [inline eth_bots/README.md:31]:
```suggestion
    >Elf-simulations requires python 3.10. We recommend Pyenv to easily manage multiple python versions, which you can install with the “[automatic installer](https://github.com/pyenv/pyenv#automatic-installer)” instructions, which tells you to run `curl https://pyenv.run | bash` in your AWS terminal.
```

File: eth_bots/README.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    >Per the output of that command when the install is finished, you’ll want to add these lines to the bottom of your `~/.bashrc`:
```
Comment (wakamex) [inline eth_bots/README.md:32]:
```suggestion
    >[Add Pyenv to your shell](https://github.com/pyenv/pyenv#set-up-your-shell-environment-for-pyenv). If using bash, that means adding these lines to the bottom of your `~/.bashrc`:
```

File: eth_bots/README.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+1. make a fork of the [delvtech/elf-simulations repo](https://github.com/delvtech/elf-simulations) ([GitHub fork instructions](https://docs.github.com/en/get-started/quickstart/fork-a-repo?tool=webui&platform=mac)).
+
+2. Install elfpy by following the installation instructions found [on github](https://github.com/delvtech/elf-simulations/blob/main/INSTALL.md).
+
+    >**💡NOTE:**
+    >
+    >**pyenv install tips:**
+    >
+    >If your using an AWS server running bash on Linux, then to install Pyenv you’ll want to follow the [automatic installer](https://github.com/pyenv/pyenv#automatic-installer) instructions, and then [add pyenv to your shell environment](https://github.com/pyenv/pyenv#set-up-your-shell-environment-for-pyenv).
```
Comment (wakamex) [inline eth_bots/README.md:16]:
```suggestion
    >If you're using an AWS server running bash on Linux, then to install Pyenv you’ll want to follow the [automatic installer](https://github.com/pyenv/pyenv#automatic-installer) instructions, and then [add pyenv to your shell environment](https://github.com/pyenv/pyenv#set-up-your-shell-environment-for-pyenv).
```

File: eth_bots/README.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+    >
+    >**git clone the correct repo:**
+    >
+    >You’ll want to clone your fork in step 2 of the install instructions.
```
Comment (wakamex) [inline eth_bots/README.md:20]:
```suggestion
    >You’ll want to clone your fork
```

File: eth_bots/README.md (hunk 6 above)
Comment (dpaiton) [reply eth_bots/README.md:16]: good catch!

**Review by wakamex — APPROVED** [review]
looks good

**Pull request merged**